src/groupeddataframe/grouping.jl

```diff
@@ -376,11 +376,16 @@ function aggregate(d::AbstractDataFrame,
     aggregate(groupby(d, cols, sort=sort), fs)
 end
 
+function dfnameof(f)
```

Comment:
Why `df`? Sounds like "data frame".

Reply:
This is the reason because I want to get a name of `f` for internal DataFrames.jl purposes. Do you have any suggestion what would be a good name.

I also can change it to an anonymous function call in line 385 if this is preferred.

Reply:
Maybe `funname` or something like that? Just bikeshedding.

Reply:
fixed